schema/schema.sql

```diff
@@ -0,0 +1,63 @@
+CREATE DATABASE catalog;
+
+CREATE TABLE items (
+  id BIGINT PRIMARY KEY,
+  genre_id BIGINT NOT NULL,
+  author_id BIGINT NOT NULL,
+  label_id BIGINT NOT NULL,
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL,
+  FOREIGN KEY (genre_id) REFERENCES genres(id),
+  FOREIGN KEY (author_id) REFERENCES authors(id),
+  FOREIGN KEY (label_id) REFERENCES labels(id)
+);
```

Comment:
- [OPTIONAL] I know it was not explicitly mentioned in the project requirement, but it would be nice to create indexes on all foreign keys in this schema based on the requirement. Creating indexes on foreign keys makes queries more efficient. 👍